pkg/plugin/repository/builtins.go

```diff
@@ -0,0 +1,12 @@
+package repository
+
+// DefaultPluginRepository is the default Draft core plugins.
+const DefaultPluginRepository = "github.com/bacongobbler/draft-plugins"
```

Comment:
TODO: switch to `github.com/draftcreate/plugins` if this is an acceptable path forward